modules/apps/29-fee/fee_test.go

```diff
@@ -19,14 +19,17 @@ type FeeTestSuite struct {
 
 	chainA *ibctesting.TestChain
 	chainB *ibctesting.TestChain
+	chainC *ibctesting.TestChain
 
-	path *ibctesting.Path
+	path  *ibctesting.Path
+	path2 *ibctesting.Path
```

Comment:
nit: want to call this `pathAToC` maybe? Don't feel too strongly but might be easier to reason about when revisiting

Reply:
updated :+1: